lib/src/run.dart

```diff
@@ -95,12 +111,14 @@ Future<String> runAsync(String executable,
     broadcastStderr.listen((List<int> data) => stderr.addAll(data));
     stderrLines.listen(logStderr);
 
+    var encoding = runOptions.stdoutEncoding != null
+        ? runOptions.stdoutEncoding : SYSTEM_ENCODING;
     return process.exitCode.then((int code) {
-      var stdoutString = SYSTEM_ENCODING.decode(stdout);
+      var stdoutString = encoding.decode(stdout);
```

Comment:
I didn't remove `stdoutEncoding` and `stderrEncoding`. Didn't see this comment when I added `done`, but I modified `toLineStream()` to use the encoding set in `RunOptions`. I can imagine this being more useful outside the english speaking area.